core/assets/bundles/bundle_fr.properties

```
@@ -185,14 +185,14 @@ mod.folder.missing = Seuls les mods sous forme de dossiers peuvent être publié
 mod.scripts.disable = Votre appareil ne prend pas en charge les mods avec des scripts. Vous devez désactiver ces mods pour pouvoir jouer.
 
 mod.dependencies.error = [scarlet]Mods are missing dependencies
-mod.dependencies.soft = (optional)
+mod.dependencies.soft = (optionnel)
 mod.dependencies.download = Import
-mod.dependencies.downloadreq = Import Required
-mod.dependencies.downloadall = Import All
-mod.dependencies.status = Import Results
-mod.dependencies.success = Successfully downloaded:
-mod.dependencies.failure = Failed to download:
-mod.dependencies.imported = This mod requires dependencies. Download?
+mod.dependencies.downloadreq = Importer les requis
+mod.dependencies.downloadall = Importer les tous
+mod.dependencies.status = Résultat de l'import
+mod.dependencies.success = Téléchargement réussi :
+mod.dependencies.failure = Téléchargement en échec :
+mod.dependencies.imported = Ce mod requiert des dépendances. Voulez-vous les télécharger ?
```

```suggestion
mod.dependencies.success = Téléchargement réussi:
mod.dependencies.failure = Téléchargement en échec:
mod.dependencies.imported = Ce mod requiert des dépendances. Voulez-vous les télécharger?
```
Même chose qu'en haut.